pkg/operator/clusterlink/agent/manifests.go

```diff
@@ -15,66 +15,66 @@ spec:
       labels:
         app: clusterlink-agent
     spec:
-      tolerations:
-      - key: node-role.kubernetes.io/control-plane
-        operator: Exists
-        effect: NoSchedule
-      - key: node-role.kubernetes.io/master
-        operator: Exists
-        effect: NoSchedule
+      containers:
+        - name: clusterlink-agent
+          securityContext:
+            privileged: true
+          image: {{ .ImageRepository }}/clusterlink-agent:{{ .Version }}
+          imagePullPolicy: IfNotPresent
+          command:
+            - clusterlink-agent
+            - --kubeconfig=/etc/clusterlink/kubeconfig
+          env:
+            - name: CLUSTER_NAME
+              value: "{{ .ClusterName }}"  
+            - name: NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+          resources:
+            limits:
+              memory: 200Mi
```

Comment:
clusterlink-agent memory limits or requests does not seem to be high. Does it need to be adjusted? What do you think? @hanweisen